### ISSUE TITLE
Demonic UI

### ISSUE THREAD

**Issue by basxto** [sanctuary/ember#1]
Might be interesting to run this with http://flarerpg.org/index.php/mods/demonic-ui/

**Comment by mewmew** [comment]
@basxto Oh, that's great!! Thanks for bringing it to our attention :) Are you working on Flare too?

Cheers,
Robin

**Comment by basxto** [comment]
No, I was looking for diablo remake that is able to also run flare mods and found this instead.
And just checked out the mod section of flare recently.
